content/ko/examples/controllers/job.yaml

```
@@ -7,7 +7,7 @@ spec:
     spec:
       containers:
       - name: pi
-        image: perl
+        image: perl:5.34
```

한 가지 더 제안을 드립니다 😅

이것도 마찬가지로 선반영 사안입니다.
https://github.com/kubernetes/website/blob/main/content/en/examples/controllers/job.yaml

이 사항을 반영하실 때, 혹시 squash도 가능하시면 squash도 해 주시고,
어렵거나 하시다면 제가 label 추가하여 squash merge 하겠습니다~~

```suggestion
        image: perl:5.34.0
```